### PR TITLE
[MINOR] Fix sensitive data trimming from exceptions

### PR DESCRIPTION
**Pull request by mukkachaitanya**
## Problem

#1266 added a feature to trim sensitive data, but it missed hadling individual exceptions and only handled Nested

## Solution

Trim sensitive data before iterating over the individual exceptions

##### Does this solution apply anywhere else?
- [ ] yes
- [ ] no

##### Testing done:
- [ ] Unit tests
- [ ] Integration tests
- [ ] System tests
- [ ] Manual tests

### CODE REVIEW

File: src/test/java/io/confluent/connect/jdbc/util/LogUtilTest.java [hunk 1]
```diff
@@ -146,6 +147,28 @@ public void testSecondLevelNestedBatchUpdateSensitiveNoDetails() {
...
+    SQLException expectedTrimmed = new SQLException("e1");
+    BatchUpdateException e3 = new BatchUpdateException("Batch entry 0 INSERT INTO \"abc\" (\"c1\",\"c2\",\"c3\",\"c4\")",
+        new int[0]);
+    expectedTrimmed.setNextException(e3);
+
+    SQLException actualTrimmed = LogUtil.trimSensitiveData(e1);
+    assertEqualsSQLException(expectedTrimmed, actualTrimmed);
+  }
+
+
```
Comment (mukkachaitanya) [inline src/test/java/io/confluent/connect/jdbc/util/LogUtilTest.java:171]:
```suggestion

```

File: src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java [hunk 2]
```diff
@@ -117,7 +118,7 @@ public void put(Collection<SinkRecord> records) {
                   + "For complete details on each exception, please enable DEBUG logging.",
               totalExceptions);
           int exceptionCount = 1;
-          for (Throwable e : sqle) {
+          for (Throwable e : trimmedException) {
             log.debug("Exception {}:", exceptionCount++, LogUtil.trimSensitiveData(e));
```
Comment (ManasjyotiSharma) [inline src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java:122]: trimmig is not needed in line 122.

**Comment by ManasjyotiSharma** [comment]
LGTM @mukkachaitanya. Left one minor comment.

File: src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java (hunk 2 above)
Comment (mukkachaitanya) [reply src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java:122]:
```suggestion
            log.debug("Exception {}:", exceptionCount++, e);
```
Ack!

**Pull request merged**